components/inputs/test/input-checkbox.html

```diff
@@ -113,9 +113,30 @@
 						expect(input.indeterminate).to.be.true;
 					});
 
+					it('should bind "checked" attribute to input property', async() => {
+						elem.setAttribute('checked', 'checked');
+						await elem.updateComplete;
+						expect(input.checked).to.be.true;
+					});
+
+					it('should bind "checked" property to input property', async() => {
+						elem.checked = true;
+						await elem.updateComplete;
+						expect(input.checked).to.be.true;
+					});
+
+					it('should continue to bind "checked" property after interaction', async() => {
+						input.click();
+						expect(input.checked).to.be.true;
+						expect(elem.checked).to.be.true;
+						await elem.updateComplete;
+						elem.checked = false;
```

Comment:
Linting is failing on this with a [require-atomic-updates](https://eslint.org/docs/rules/require-atomic-updates) error. I'm confused by it, but looking into it...